src/Sodaq_dataflash.cpp

```diff
@@ -62,6 +62,9 @@ void Sodaq_Dataflash::init(uint8_t csPin)
   _pageAddrShift = 1;
 #elif DF_VARIANT == DF_AT45DB161D
   _pageAddrShift = 1;
+// Adding AT45DB041D:
```

Comment:
Can you drop these comments "Adding ..." (three times). The other variants don't have it either.